lib/chef/resources.rb

```diff
@@ -73,6 +73,8 @@
 require_relative "resource/homebrew_tap"
 require_relative "resource/homebrew_update"
 require_relative "resource/ifconfig"
+require_relative "resource/inspec_input"
+require_relative "resource/inspec_waiver"
 require_relative "resource/inspec_waiver_file_entry"
```

Comment:
Is this enough different to keep it around or should we deprecate this resource?

Reply:
should probably get deprecated, but i don't entirely understand what it is for